### DISCUSSION TITLE
List all keys

### DISCUSSION THREAD

**Discussion by AdrKacz** [node-config/node-config#748] [Q&A] [answered]
How would you list all `keys` available in the `config` object under `get/has`?

**Comment by jdmarshall** [comment, marked as answer]
I know I tried lodash for this but I believe I settled on the traverse package to handle this, since traverse gives you a way to build the canonical key path when you hit the leaves. And then you can use lodash.get to walk the path. 

https://www.npmjs.com/package/traverse
https://github.com/ljharb/js-traverse?tab=readme-ov-file#paths
